bower.json

```diff
@@ -21,7 +21,7 @@
     "npm-debug.log"
   ],
   "dependencies": {
-    "polymer": "^1.7.0",
+    "polymer": "^1.10.1",
```

Comment:
Please revert this, this will force other consumers of the design system to pick this version of Polymer.